dev/test/write-batch.ts

```diff
@@ -66,10 +66,6 @@ describe('set() method', () => {
     );
   });
 
-  it('accepts preconditions', () => {
```

Comment:
writeBatch.set() does not accept preconditions

Reply:
But it does accept them as document data :) Thanks for fixing.

Reply:
updated the test name